lib/gradebook.rb

```diff
@@ -1 +1,26 @@
+class Gradebook
+  attr_reader :instructor,
+              :courses
 
+  def initialize(instructor)
+    @instructor = instructor
+    @courses = []
+    @students_below = []
+  end
+
+  def add_course(course)
+    @courses << course
+  end
+
+  def students_below
+    @courses.each do |course|
+      students = course.students
+      students.each do |student|
+        if student.grade <= 60
+          @students_below << student
+        end
+      end
+    end
+    @students_below
+  end
```

Comment:
This method should take an argument for a dynamic threshold: 
```suggestion
  def students_below(threshold)
    @courses.each do |course|
      students = course.students
      students_below_threshold = []
      students.each do |student|
        if student.grade <= threshold
           students_below << student
        end
      end
    end
     students_below_threshold
  end
```
I also don't think there's much value in storing the return value of this method in an instance variable.